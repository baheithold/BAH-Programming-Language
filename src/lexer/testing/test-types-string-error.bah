# This file contains an error


"   this is a string   "
"this is a string
"

"this
is a
string\" just
testing\"
