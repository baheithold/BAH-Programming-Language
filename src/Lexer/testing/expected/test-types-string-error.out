STRING_TYPE "   this is a string   "
STRING_TYPE "this is a string
"
ERROR, line 12: BAD_STRING
